src/connector.js

```diff
@@ -63,10 +67,8 @@ class BotiumConnectorSimpleSocketIO {
     if (this.caps[Capabilities.SIMPLESOCKETIO_ENDPOINTPATH]) {
       this.socketOptions.path = this.caps[Capabilities.SIMPLESOCKETIO_ENDPOINTPATH]
     }
-    if (this.caps[Capabilities.SIMPLESOCKETIO_AUTH_TOKEN]) {
-      this.socketOptions.auth = {
-        token: this.caps[Capabilities.SIMPLESOCKETIO_AUTH_TOKEN]
-      }
+    if (this.caps[Capabilities.SIMPLESOCKETIO_CLIENT_OPTIONS]) {
+      this.socketOptions = Object.assign({}, this.caps[Capabilities.SIMPLESOCKETIO_CLIENT_OPTIONS], this.socketOptions)
```

Comment:
Usually we are doing this in the "opposite" direction - to make it possible to overwrite event the socket options set internally, the order should be reversed here.

Reply:
fixed